e2e-tests/tfm/cypress/e2e/journeys/case-amendments/facility-end-date/feature-flag-disabled/gef-deal-expect-no-facility-end-date-features.spec.js

```diff
@@ -0,0 +1,85 @@
+import relative from '../../../../relativeURL';
+import facilityPage from '../../../../pages/facilityPage';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1, T1_USER_1 } from '../../../../../../../e2e-fixtures';
+import { MOCK_APPLICATION_AIN } from '../../../../../fixtures/mock-gef-deals';
+import { MOCK_FACILITY_ONE } from '../../../../../fixtures/mock-gef-facilities';
+import { DEAL_TYPE } from '../../../../../../../gef/cypress/fixtures/constants';
+import amendmentsPage from '../../../../pages/amendments/amendmentsPage';
+import dateConstants from '../../../../../../../e2e-fixtures/dateConstants';
+
+if (Cypress.env('FF_TFM_FACILITY_END_DATE_ENABLED') === 'false') {
+  context('Amendments - GEF deal does not display any Facility end date pages or fields when TFM feature flag disabled', () => {
+    let dealId;
+    let dealFacilities;
+
+    const MOCK_GEF_FACILITY_WITH_FACILITY_END_DATE = {
+      ...MOCK_FACILITY_ONE,
+      hasBeenIssued: true,
+      isUsingFacilityEndDate: true,
+      facilityEndDate: new Date('2023-02-01'),
+      bankReviewDate: undefined,
+    };
+
+    before(() => {
+      // inserts a gef deal
+      cy.insertOneGefDeal(MOCK_APPLICATION_AIN, BANK1_MAKER1).then((insertedDeal) => {
+        dealId = insertedDeal._id;
+        // updates a gef deal so has relevant fields
+        cy.updateGefDeal(dealId, MOCK_APPLICATION_AIN, BANK1_MAKER1);
+
+        cy.createGefFacilities(dealId, [MOCK_GEF_FACILITY_WITH_FACILITY_END_DATE], BANK1_MAKER1).then((createdFacilities) => {
+          dealFacilities = createdFacilities.details;
+        });
+
+        cy.submitDeal(dealId, DEAL_TYPE.GEF, T1_USER_1);
+      });
+    });
+
+    beforeEach(() => {
+      cy.login(PIM_USER_1);
+      cy.visit(relative(`/case/${dealId}/facility/${dealFacilities._id}`));
+    });
+
+    after(() => {
+      cy.deleteDeals(dealId, ADMIN);
```

Comment:
I know its legacy, but does this only delete one deal?

Reply:
Definitely beyond scope to change this

Reply:
It does just delete one deal despite the name 😭 But yes, won't change here